src/common_interface/integrator_types.jl

```diff
@@ -147,42 +147,42 @@ DiffEqBase.check_error(integrator::AbstractSundialsIntegrator) =
     interpret_sundials_retcode(integrator.flag)
 
 DiffEqBase.postamble!(integrator::AbstractSundialsIntegrator) = nothing
-# No-op postamble! to make check_error! (and hence iterator interface
+# No-op postamble! to make DiffEqBase.check_error! (and hence iterator interface
 # implemented in DiffEqBase) work.
 
 ### Iterator interface
 
 @inline function DiffEqBase.step!(integrator::AbstractSundialsIntegrator)
   if integrator.opts.advance_to_tstop
-    while integrator.tdir*(integrator.t-top(integrator.opts.tstops)) < -1e6eps()
-        tstop = top(integrator.opts.tstops)
+    while integrator.tdir*(integrator.t-DiffEqBase.top(integrator.opts.tstops)) < -1e6eps()
```

Comment:
not from DiffEqBase. It's from DataStructures.jl

Reply:
Sorry, I will fix it

Reply:
fixed via 2b52369